modules/installation-aws-add-local-zone-locations.adoc

```diff
@@ -36,6 +54,7 @@ Depending on the AWS Region, the list of available zones can be long. The comman
 [source,terminal]
 ----
 $ aws ec2 modify-availability-zone-group \
-    --group-name "<value_of_GroupName>" \
+    --group-name "<value_of_GroupName>" \// <1>
```

Comment:
Docs team check: Check callouts that targets commands that span multiple lines works on Customer Portal.

Reply:
You'll need to validate this after the content is merged.